velox/functions/prestosql/types/TimestampWithTimeZoneType.h

```diff
@@ -125,4 +124,24 @@ class TimestampWithTimeZoneTypeFactories : public CustomTypeFactories {
 
 void registerTimestampWithTimeZoneType();
 
+using TimeZoneKey = int16_t;
+
+static constexpr int32_t kTimezoneMask = 0xFFF;
+static constexpr int32_t kMillisShift = 12;
```

Comment:
nit
static should not be used for globals in a header.
For these two constexpr would suffice.

Reply:
Already fixed.